src/main/java/pl/team/carrent/service/ClientService.java

```diff
@@ -0,0 +1,61 @@
+package pl.team.carrent.service;
+
+
+import org.springframework.stereotype.Service;
+import pl.team.carrent.client.ClientNotExistException;
+import pl.team.carrent.model.Client;
+import pl.team.carrent.repository.ClientRepository;
+import pl.team.carrent.client.SearchClientOption;
+
+import java.util.HashSet;
+import java.util.Set;
+
+
+/**
+ * @author: Jakub O.  [https://github.com/JacobTheLiar]
+ * @date : 2019-11-09 10:11
+ * *
+ * @className: ClientService
+ * *
+ * *
+ ******************************************************/
+@Service
+public class ClientService{
+    
+    private ClientRepository clientRepository;

```

Comment:
Chcemy, żeby to było final, aby podczas pracy aplikacji nikt nie podmienił nam tego repo na nulla np